README.md

```diff
@@ -2,7 +2,13 @@
 
 # How to Run PyTorch with TPUs
 
-First, create your [TPU](https://pantheon.corp.google.com/compute/tpus) node with the corresponding release you wish to consume (TPU software version: ex. `pytorch-0.5`):
+You can run your models on [Colab](https://github.com/pytorch/xla/tree/master/contrib/colab) or you can create your own Compute VM and Cloud TPU pair (better for performance and serious training). We also have tutorials available on Google Cloud website:
```

Comment:
I would honestly put those at the bottom ... like an "Oh, by the way, you can also ...".
Running on Colab is a bit "secondary" IMHO.

Reply:
@dlibenzi IMHO it's definitely helpful to keep colab demos on top before GCP setup since it's so much easier for users to access. 
But we probably want to keep it in a "try it out in 5s" section and highlight "Colab cannot give good performance for large model training, please switch to GCP VMs" in both readme and the colab notebook.

Reply:
IMHO .. Colab == Toy 😄 
Also, Colab == PITA when it comes to versioning and running cutting edge (like we do).
So this should go in an "Oh, by the way, ..." section below (or at the bottom of) the real setup one.

Reply:
when toys come way more handy(and free) than a 10-step setup which might also cost money ;) , people might still prefer to play with toys a bit before GCP.

Reply:
I think we can agree on disagreeing here 😄 
We will have to list a way, for which we will have to remind that performance will suck, and also that it will be broken at some point for some random XRT fix which is not being picked up by an older TF release we run on the TPU VM side.
This calls for a "secondary listing" instead of primary.

Reply:
Let me know if the "secondary listing" of colab this way works.